tests/unit/features/RandomFourierDotFeatures_unittest.cc

```diff
@@ -176,3 +176,58 @@ TEST(RandomFourierDotFeatures, add_to_dense_test)
 
 }
 
+TEST(RandomFourierDotFeatures, gauss_coeff_dim)
+{
+	int32_t num_dims = 50;
+	int32_t num_vecs = 5;
+	int32_t D = 100;
```

Comment:
could you name this num_dims_rff or something that is less confusing?

Reply:
why resolved?

Reply:
this is still not resolved. Pls just don't mark conversations as resolved....it doesn't really help

Reply:
Really sorry!. By the way, I changed the name to ```num_dims_embedding``` I guess that wouldn't do?

Reply:
+1 But here you have still "D"

Reply:
problem, every subtest uses D for "dim_output" shall I change for all?

Reply:
yes I think it is a good idea

Reply:
with "yes" I meant "change all occurences....`dim_output` is not a good name, I suggested `dims_rff` or `num_dims_rff` above

Reply:
Right, misunderstanding from my part. So I'll name the test "dim_output"  and the "D" variable as "num_dims_rff". All is well?